ondemandnavi: Update permissions

Add the new display and audio permissions required with the change to
running as non-root.

Upstream-Status: pending

Signed-off-by: Scott Murray <scott.murray@konsulko.com>

diff --git a/package/config.xml b/package/config.xml
index ae5c1f6..10b2d99 100755
--- a/package/config.xml
+++ b/package/config.xml
@@ -13,6 +13,8 @@
   <feature name="urn:AGL:widget:required-permission">
     <param name="urn:AGL:permission::public:no-htdocs" value="required"/>
     <param name="http://tizen.org/privilege/internal/dbus" value="required"/>
+    <param name="urn:AGL:permission::public:display" value="required" />
+    <param name="urn:AGL:permission::public:audio" value="required" />
   </feature>
 </widget>
 
